.changeset/thirty-drinks-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
```

Comment:
Not sure if it's major, isn't a patch although we're in pre-release?

Reply:
Ok make sense, also make sure it'll be picked up by the changeset versioning, which I think it will